string world_sdf_filepath
int8 num_free_positions
---
geometry_msgs/Point[] free_positions
